.github/workflows/CI.yml

```diff
@@ -18,6 +18,9 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+      - name: Update npm
+        run: npm i -g npm@8.19.4
+
```

Comment:
I'm curious why this is necessary if it's the only change in the file.

Reply:
The old version of `npm` bundled with node 12 & 14 doesn't do recursive `.bin` lookups (apparently). So to avoid needing relative paths in the test scripts, I just updated `npm`, since it's not what's being tested :)